data/kz/4.0.affix

```diff
@@ -2,7 +2,7 @@
 "(" "$" "``": LPUNC+;
 
 %dakileri da ki ler i leri kileri düm di lar diler um yor mi ni i in diğ ın diğini ini yorum miyorum: SUF+;
-ге дым ым ім ты ті ды ді м дім дар тер тар тардың лар лер дер  гісі ген гің ін генін ме ймін меймін лер да ғы лар ды ларды ғыларды дағыларды ның нің дың дің тың тің: SUF+;
+дым ым ім ны ні ты ті ды ді м дім дар тер тар тардың лар лер дер  гісі ген гің ін генін ме ймін меймін лер ғы лар ды ларды ғыларды дағыларды ның нің дың дің тың тің ға ге қа ке мен бен пен да де та те нан нен дан ден тан тен: SUF+;
```

Comment:
It's OK to have these on multiple lines; there is no requirement that it be one very long line.